CHANGELOG.md

```diff
@@ -245,7 +249,7 @@
 * Adjust curl retry and connection timeout handling
 * Switch to the recommended regional S3 domain instead of the global one
 
-## [v163] - 2022-06-09), v167 (published by mistake 2022-09-08
+## [v163] - 2022-06-09, [v167] (published by mistake 2022-09-08)
```

Comment:
Since `v167` had no link in the body, my editor thought the link table didn't need a `v167`.